projects/lib/src/board/westernboard.cpp

```diff
@@ -851,7 +851,12 @@ void WesternBoard::setCastlingSquare(Side side,
 		xorKey(m_zobrist->castling(side, rs));
 	if (square != 0)
 		xorKey(m_zobrist->castling(side, square));
-	rs = square;
```

Comment:
formatting, these 3 lines should not be patched

Reply:
This is strange, I have never change the code before.